server/svix-server/src/webhook.rs

```diff
@@ -0,0 +1,298 @@
+// SPDX-FileCopyrightText: © 2022 Svix Authors
```

Comment:
We are going to move it to its own library (like the other libs) after you're done, so just keep that in mind when you design things.